.github/workflows/release-workflow.yml

```diff
@@ -59,11 +59,17 @@ jobs:
           aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: us-west-2
+
       - name: Upload Artifacts to S3
         run: |
-          s3_path=s3://artifacts.opendistroforelasticsearch.amazon.com/downloads
-          aws s3 cp ${{ steps.build_zip.outputs.zip_path }} $s3_path/kibana-plugins/opendistro-alerting/
-          aws cloudfront create-invalidation --distribution-id ${{ secrets.DISTRIBUTION_ID }} --paths '/downloads/*'
+          zip=${{ steps.build_zip.outputs.zip_path }}
+          zip_outfile=`basename ${zip%.zip}-build-${GITHUB_RUN_NUMBER}.zip`
+
+          s3_prefix="s3://staging.artifacts.opendistroforelasticsearch.amazon.com/snapshot/kibana-plugins/alerting/"
+
+          echo "Copying ${zip} to ${s3_prefix}${zip_outfile}"
+          aws s3 cp --quiet $zip ${s3_prefix}${zip_outfile}
```

Comment:
Is this being copied into **${zip_outfile}** or copied as (named) **${zip_outfile}**

Reply:
It's being copied into a file called ${zip_outfile} in S3, where ${zip_outfile} is the basename of the zip artifact with a build number injected.

For example, if the original zip is `/path/to/alerting-kibana-1.0.zip` then it will create a file in S3 called `/snapshot/kibana-plugins/alerting/alerting-kibana-1.0-build-27.zip`

Reply:
Got it! Thanks!